respa_admin/templates/respa_admin/resources/_unit_list.html

```diff
@@ -0,0 +1,47 @@
+{% load i18n %}
+{% load thumbnail %}
+
+<section class="units">
+    <div class="container">
+
+        <div class="filter-bar container-fluid">
+            <h2 class="pull-left">{% trans "All units" %}</h2>
+        </div>
+
+        <div class="unit-list container-fluid">
+            <ul class="list-group">
+                <li class="list-group-item sort-bar">
+                    <div class="pull-left empty-holder col-1"></div>
+                    <a class="col-2 btn order-by-link" href="?order_by={% if order_by == "name" %}-name{% else %}name{% endif %}">
+                        {% trans "Name" %} <span class="glyphicon glyphicon-sort" aria-hidden="true"></span>
+                    </a>
+                    <span class="col-3 order-by-link">
```

Comment:
Should users be able to order results by `editable`-status?

Reply:
That's a good question. I didn't figure that would be a useful usecase. Maybe @petterill can weigh in on this one?

Reply:
Probably users are interested in editing a specific unit and not so much interested in all units that can be edited. So just having them in alphabetical order is most useful. No need to sort by editable status at this time.